multi-vterm.el

```diff
@@ -167,7 +174,9 @@ Optional argument DEDICATED-WINDOW: There are three types of DEDICATED-WINDOW: d
 (defun multi-vterm-project-get-buffer-name ()
   "Get project buffer name."
   (multi-vterm-format-buffer-name
-   (project-root (or (project-current) `(transient . ,default-directory)))))
+   (if multi-vterm-projectile-installed-p
```

Comment:
Can you extract this code into a function `multi-vterm-project-root` and similar to `multi-vterm-project-p`?

Reply:
Yes, that's a great idea, I can definitely do that.